impl/graphsync_test.go

```diff
@@ -69,130 +66,6 @@ var protocolsForTest = map[string]struct {
 	"(v1.0 -> v1.0)": {[]protocol.ID{gsnet.ProtocolGraphsync_1_0_0}, []protocol.ID{gsnet.ProtocolGraphsync_1_0_0}},
 }
 
-func TestMakeRequestToNetwork(t *testing.T) {
```

Comment:
my guess is that these two are removed because they were, as you were saying recently, early bootstrapping tests that made sense before the whole thing was built, now they are a bit too basic to be bothered with?

Reply:
yep